app/assets/javascripts/map.js

```diff
@@ -0,0 +1,210 @@
+//= require polyline
+var map;
+
+var directionsService = new google.maps.DirectionsService();
+
+function initialize() {
```

Comment:
This should really be part of an object -- at the very least a method off an object literal -- so you don't have a very generically named method `initialize` in the global namespace just waiting to be overwritten. :smiling_imp: